src/views/pages/index.ejs

```diff
@@ -21,26 +20,74 @@
       <div class="flex flex-1 flex-col sm:w-1/3 text-center w-full">
-        <div class="p-5 text-3xl bg-white relative ">Created (<%= created.length %>)</div>
-        <% for(let i=0; i<created.length; i++){ %>
-        <%- include('../partials/jobCard', {status: created, current: i}) %>
-        <%}%>
+        <div class="p-5 text-3xl bg-white relative ">Created (<span id="created-length"><%= created.length %></span>)</div>
+        <div id="created" data-ajax='true' data-ajax-postfn="window.updateTitle" data-ajax-url="" data-ajax-update="#created" data-ajax-mode="replace" data-ajax-confirm="">
+          <% for(let i=0; i<created.length; i++){ %>
+            <%- include('../partials/jobCard', {job: created, current: i}) %>
+          <%}%>
+        </div>
       </div>
       <div class="flex flex-1 flex-col sm:w-1/3 text-center w-full">
-        <div class="p-5 text-3xl bg-white relative ">Pending (<%= pending.length %>)</div>
-        <% for(let i=0; i<pending.length; i++){ %>
-          <%- include('../partials/jobCard', {status: pending, current: i}) %>
-        <%}%>
+        <div class="p-5 text-3xl bg-white relative ">Pending (<span id="pending-length"><%= pending.length %></span>)</div>
+        <div id="pending" data-ajax='true' data-ajax-postfn="window.updateTitle" data-ajax-url="" data-ajax-update="#pending" data-ajax-mode="replace" data-ajax-confirm="" >
+          <% for(let i=0; i<pending.length; i++){ %>
+            <%- include('../partials/jobCard', {job: pending, current: i}) %>
+          <%}%>
+        </div>
       </div>
       <div class="flex flex-1 flex-col sm:w-1/3 text-center w-full">
-        <div class="p-5 text-3xl bg-white relative ">Running (<%= running.length %>)</div>
-        <% for(let i=0; i<running.length; i++){ %>
-        <%- include('../partials/jobCard', {status: running, current: i}) %>
-        <%}%>
+        <div class="p-5 text-3xl bg-white relative ">Running (<span id="running-length"><%= running.length %></span>)</div>
+        <div id="running" data-ajax='true' data-ajax-postfn="window.updateTitle" data-ajax-url="" data-ajax-update="#running" data-ajax-mode="replace" data-ajax-confirm="">
+          <% for(let i=0; i<running.length; i++){ %>
+            <%- include('../partials/jobCard', {job: running, current: i}) %>
+          <%}%>
+        </div>
       </div>
     </div>
     <footer>
       <%- include('../partials/footer'); %>
     </footer>
+  </div>
+  <script>
+    function buildApiUrl(columnName){
+      const apiUrl = new URL(`/api`+document.location.pathname, document.location.origin)
+      apiUrl.searchParams.append(`status`,columnName)
+      document.getElementById(columnName).setAttribute("data-ajax-url", apiUrl);
+    }
+  </script>
+   <script>
+    var userIsActive;
+    var timeout;
+    buildApiUrl("created")
+    buildApiUrl("pending")
+    buildApiUrl("running")
```

Comment:
Don't use JS to build this. If you really want a clean DRY, you generate it with html template.

However, I would recommend just explicitly hard code the url on html. Because, from my experience, it is not wise to write very DRY code on UI. UI depends on user requirement and user requirement changes a lot. DRY code is built on top of assumption, and if the assumption breaks, you will need to refactor.